go/oasis-node/cmd/node/node_control.go

```diff
@@ -175,7 +176,7 @@ func (n *Node) getIdentityStatus() control.IdentityStatus {
 	return control.IdentityStatus{
 		Node:      n.Identity.NodeSigner.Public(),
 		Consensus: n.Identity.ConsensusSigner.Public(),
-		TLS:       n.Identity.GetTLSPubKeys(),
+		TLS:       []signature.PublicKey{n.Identity.TLSSigner.Public()},
```

Comment:
Can we remove the array?

Reply:
The only downside is that it breaks compatibility. But otherwise making it consistent with the others makes sense.